templates/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM node:12.16 AS JS_BUILD
 COPY webapp /webapp
 WORKDIR webapp
-RUN npm install && npm run build --prod
+RUN npm install && npm run build:prod
```

Comment:
I think it is a bit misleading for React developers (as we already discussed this flag is redundant in React projects). But we can keep it for now.

Reply:
Okay, I agree